client/index.html

```diff
@@ -7,6 +7,14 @@
 </head>
 
 <body>
+  
+
+  <ui-view></ui-view>
+
```

Comment:
What was the reason for moving this up and adding all the white space? Wasn't it working fine before?